falcon/media/validators/jsonschema.py

```diff
@@ -8,7 +8,7 @@
     pass
 
 
-def validate(schema):
+def validate(schema_request=None, schema_response=None):
```

Comment:
Nitpick:  Would you mind swapping the word order in these arguments to follow the _Modifier-Subject_ style, i.e.: `request_schema` and `response_schema` or `req_schema` and `resp_schema`?